templates/CRM/common/paymentBlock.tpl

```diff
@@ -85,23 +85,21 @@
       {else}
         {capture assign='preProfileID'}{/capture}
       {/if}
+      {if !array_key_exists('urlPathVar', $form)}{assign var='urlPathVar' value=null}{/if}
       {if $urlPathVar}
         {capture assign='urlPathVar'}{$urlPathVar}&{/capture}
-      {else}
-        {capture assign='urlPathVar'}{/capture}
       {/if}
-      {if $billing_profile_id}
+      {if array_key_exists('billing_profile_id', $form)}
         {capture assign='profilePathVar'}billing_profile_id={$billing_profile_id}&{/capture}
       {else}
-        {capture assign='profilePathVar'}{/capture}
+        {assign var='profilePathVar' value=null}
```

Comment:
profilePath var not looking more encouraging 

![image](https://github.com/civicrm/civicrm-core/assets/336308/1ba4a9b3-225a-4de3-abf3-a8873b4c8630)

Reply:
oh that depends on `billing_profile_id`

Reply:
OK - I think this is a better fix for billing_profile_id / profile_pathVar 

https://github.com/civicrm/civicrm-core/pull/26960